mod/ArchipelagoRandomizer.csproj.user

```diff
@@ -2,4 +2,12 @@
   <PropertyGroup>
     <OutputPath>$(AppData)\OuterWildsModManager\OWML\Mods\Ixrec.ArchipelagoRandomizer</OutputPath>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Update="ArchConsoleManager.cs">
+      <SubType>Code</SubType>
+    </Compile>
+    <Compile Update="FixConsoleLayout.cs">
+      <SubType>Code</SubType>
+    </Compile>
+  </ItemGroup>
```

Comment:
What does this do which only these two files need?

Reply:
It just (tries to) make the text of the console not overlap itself. A bit of an annoying bug with Layout Groups is they don't always update immediately.

Reply:
I... don't understand what that has to do with the contents of this file. This looks like it's saying "compile this file as code", but isn't that what we already do with all the .cs files? What is actually happening in _this_ file, and how does it have anything to do with the runtime text layout?

Reply:
I don't remember editing it

Reply:
That change can probably be discarded, weird.

Reply:
Okay, so it is just weird IDE detritus. For some reason mine doesn't do this when I add files so yeah let's ditch this change.